app/helpers/facets_helper.rb

```diff
@@ -36,8 +36,7 @@ def facet_label(facet_name, facet_value = nil)
       unless ['PROVIDER', 'DATA_PROVIDER', 'MIME_TYPE', 'IMAGE_SIZE'].include?(facet_name)
         mapped_value = mapped_value.split.map(&:capitalize).join(' ')
       end
-
-      mapped_value
+      mapped_value.size  > 0 ? mapped_value : false
```

Comment:
Unnecessary spacing detected.<br>Operator `>` should be surrounded by a single space.